src/katello/agent/pulp/libdnf.py

```diff
@@ -540,8 +547,8 @@ def _download(self):
         """
         downloads = []
         for tx in self.transaction:
-            if tx.installed:
-                downloads.append(tx.installed)
+            if tx.action in [libdnf.transaction.TransactionItemAction_INSTALL, libdnf.transaction.TransactionItemAction_UPGRADE]:
```

Comment:
should we have

```python
INSTALL_ACTION_LIST = [
    libdnf.transaction.TransactionItemAction_INSTALL,
    libdnf.transaction.TransactionItemAction_UPGRADE
]
```

and use it instead of this long line?